src/components/Dropdown.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component, PropTypes } from 'react'
+
+class Dropdown extends Component {
+  static propTypes = {
+    children: PropTypes.any,
+    title: PropTypes.string,
+    options: PropTypes.array,
+    isVisible: PropTypes.bool,
+    selected: PropTypes.string,
```

Comment:
Can we also make this `PropTypes.string.isRequired` so the user of this component must provide a selected string?